src/components/plugins/export/ExportControls.jsx

```diff
@@ -29,10 +31,12 @@ class ExportControls extends Component<Props> {
     for (let i = 0; i < data.length; i += 1) {
       let rowData = "";
       for (let j = 0; j < columnModel.length; j += 1) {
-        const value = columnModel[j].formatter
-          ? columnModel[j].formatter(data[i][columnModel[j].dataIndex])
-          : data[i][columnModel[j].dataIndex];
-        rowData += `${value},`;
+        if (columnModel[j].export) {
+          const value = columnModel[j].formatter
+            ? columnModel[j].formatter(data[i][columnModel[j].dataIndex])
```

Comment:
The formatter will return a value that is displayed on the UI for this column. That can be html, a button or something with css. Not necessarily a formatted number or special field in case you are expecting that.
This breaks single responsibility principle.
We would be better off exposing an exporter function that specifically handles how the value is added to the csv being exported from the state.